regression/abstraction/test2/Readme

```diff
@@ -0,0 +1,4 @@
+Assertions with function "bar" that has hidden index variable.
+We error out currently because functions are not supported inside assertions. 
+But even when they are, we should still error as index variable doesn't appear explicitly in the
+args list and it cannot be checked using is_precise function.
```

Comment:
Rather than a readme, why not put this in the comments section of the `test.desc`?

Reply:
Sure, I can do that.

Reply:
Thanks!